packages/create-redwood-app/src/create-redwood-app.js

```diff
@@ -92,12 +102,6 @@ import { name, version } from '../package'
       type: 'boolean',
       describe: "Create even if target directory isn't empty",
     })
-    .option('telemetry', {
-      default: true,
-      type: 'boolean',
-      describe:
-        'Enables sending telemetry events for this create command and all Redwood CLI commands https://telemetry.redwoodjs.com',
-    })
```

Comment:
Using the suggestion in another comment, you can leave this here. Solves the documentation problem!